src/app/fdctl/config.c

```diff
@@ -469,220 +470,365 @@ username_to_uid( char * username ) {
-  /**/                                                      TILE_OUT( "poh",    0UL, "crds_shred",   0UL    );
+  /**/                                                      fd_topob_tile_out( topo, "poh",    0UL,                        "gossip_pack",  0UL                                );
+  /**/                                                      fd_topob_tile_out( topo, "poh",    0UL,                        "stake_out",    0UL                                );
+  /**/                                                      fd_topob_tile_out( topo, "poh",    0UL,                        "crds_shred",   0UL                                );
+
+  /* There is a special fseq that sits between the pack, bank, an poh
+     tiles to indicate when the bank/poh tiles are done processing a
+     microblock.  Pack uses this to determine when to "unlock" accounts
+     that it marked as locked because they were being used. */
+
+  for( ulong i=0UL; i<config->layout.bank_tile_count; i++ ) {
+    fd_topo_obj_t * busy_obj = fd_topob_obj( topo, "fseq", "bank_busy" );
+
+    fd_topo_tile_t * bank_tile = &topo->tiles[ fd_topo_find_tile( topo, "bank", i ) ];
+    fd_topo_tile_t * poh_tile = &topo->tiles[ fd_topo_find_tile( topo, "poh", 0UL ) ];
+    fd_topo_tile_t * pack_tile = &topo->tiles[ fd_topo_find_tile( topo, "pack", 0UL ) ];
+    fd_topob_tile_uses( topo, bank_tile, busy_obj, FD_SHMEM_JOIN_MODE_READ_WRITE );
+    fd_topob_tile_uses( topo, poh_tile, busy_obj, FD_SHMEM_JOIN_MODE_READ_WRITE );
+    fd_topob_tile_uses( topo, pack_tile, busy_obj, FD_SHMEM_JOIN_MODE_READ_ONLY );
+
+    char path[ 64 ];
+    FD_TEST( fd_cstr_printf_check( path, sizeof(path), NULL, "bank_busy.%lu", i ) );
+    FD_TEST( fd_pod_insert_ulong( topo->props, path, busy_obj->id ) );
+  }
+
+  /* There's another special fseq that's used to communicate the shred
+     version from the Solana Labs boot path to the shred tile. */
+  fd_topo_obj_t * poh_shred_obj = fd_topob_obj( topo, "fseq", "poh_shred" );
+  fd_topo_tile_t * poh_tile = &topo->tiles[ fd_topo_find_tile( topo, "poh", 0UL ) ];
+  fd_topo_tile_t * shred_tile = &topo->tiles[ fd_topo_find_tile( topo, "shred", 0UL ) ];
+  fd_topob_tile_uses( topo, poh_tile, poh_shred_obj, FD_SHMEM_JOIN_MODE_READ_WRITE );
+  fd_topob_tile_uses( topo, shred_tile, poh_shred_obj, FD_SHMEM_JOIN_MODE_READ_ONLY );
+
+  char path[ 64UL ];
+  FD_TEST( fd_cstr_printf_check( path, sizeof(path), NULL, "poh_shred" ) );
+  FD_TEST( fd_pod_insert_ulong( topo->props, path, poh_shred_obj->id ) );
+
+  for( ulong i=0UL; i<topo->tile_cnt; i++ ) {
+    fd_topo_tile_t * tile = &topo->tiles[ i ];
+    if( FD_UNLIKELY( !strcmp( tile->name, "net" ) ) ) {
+      strncpy( tile->net.app_name, config->name, sizeof(tile->net.app_name) );
+      strncpy( tile->net.interface, config->tiles.net.interface, sizeof(tile->net.interface) );
+      tile->net.xdp_aio_depth = config->tiles.net.xdp_aio_depth;
```

Comment:
I know this is basically copied, but would you mind cleaning it up? I think adding blank lines in between the cases and aligning equal signs will help.